test/Hog.C

```diff
@@ -139,12 +139,12 @@ public:
 
       // IO redirect
       int ofd  = dup(STDOUT_FILENO);
-      int nofd = open("/dev/null", O_WRONLY);
+      int nofd = open("stdout", O_WRONLY|O_APPEND|O_CREAT, 00644);
```

Comment:
Maybe we just disable this code instead of duplicating stdout and setting it to itself?